### PR TITLE
Add CSV, CRD, and changelog for v1.1.0

### PR DESCRIPTION
**Pull request by kurtismullins**
Changes
- Create CSV and CRD for v1.1.0
- Adds Changelog starting with v1.1.0
- Uses Project Quay in place of Red Hat Quay in CSV

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+1.1.0
+
+- Add additional roles to CSV to manage ingresses. #202
+- Always use Port 8443 for Quay Config App's health probes. #200
+- (Not sure? Prometheus something something) #197
+- Add TLS Configuration for Clair's Postgres connection. #184
```
Comment (kurtismullins) [inline CHANGELOG.md]: Some of these may not be part of 1.1.0. We need to remove those which are not.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- Always use Port 8443 for Quay Config App's health probes. #200
+- The Quay Config App now continues running by default. #189
+- The Redis and Hostname configuration are marked "Read Only" in the Quay
+  Configuration App. #188
+- The Quay Config App can now be used successfully to manage Quay's
+  configuration. #187
+- The "Repo Mirror" pod is now health-checked using the correct port. #187
+- Multiple superusers can now be declared. #187
+- Added support for injecting config files for Quay and Clair. #187
+- (OpenShift) SCC is now cleaned up when the QuayEcosystem is deleted. #187
```
[the remark this answers is not in the document]
Comment (kurtismullins) [reply CHANGELOG.md]: @sabre1041 Same PR?

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- Certificates and other secrets are now mounted in a way that is compatible
```
[the remark this answers is not in the document]
Comment (kurtismullins) [reply CHANGELOG.md:39]: @sabre1041 "managed Quay certificates natively" -- what does that mean?
Also, which PR? I just copied & pasted this from the Google doc we used to collaborate

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+- Add additional roles to CSV to manage ingresses. #202
+- Always use Port 8443 for Quay Config App's health probes. #200
+- The Quay Config App now continues running by default. #189
+- The Redis and Hostname configuration are marked "Read Only" in the Quay
+  Configuration App. #188
+- The Quay Config App can now be used successfully to manage Quay's
+  configuration. #187
+- The "Repo Mirror" pod is now health-checked using the correct port. #187
+- Multiple superusers can now be declared. #187
+- Added support for injecting config files for Quay and Clair. #187
```
[the remark this answers is not in the document]
Comment (kurtismullins) [reply CHANGELOG.md]: Good point. I can change the wording there.

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+1.1.0
+
+- Add additional roles to CSV to manage ingresses. #202
+- Always use Port 8443 for Quay Config App's health probes. #200
+- The Quay Config App now continues running by default. #189
+- The Redis and Hostname configuration are marked "Read Only" in the Quay
+  Configuration App. #188
+- The Quay Config App can now be used successfully to manage Quay's
```
[the remark this answers is not in the document]
Comment (kurtismullins) [reply CHANGELOG.md]: Relatively, yes. It is the way the secret had a mount directly on it which broke Quay's ability to read/write the secret as expected.

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  configuration. #187
+- The "Repo Mirror" pod is now health-checked using the correct port. #187
+- Multiple superusers can now be declared. #187
```
[the remark this answers is not in the document]
Comment (kurtismullins) [reply CHANGELOG.md]: Good point. I'll clean up the wording on this one too.

File: CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+- (OpenShift) SCC is now cleaned up when the QuayEcosystem is deleted. #187
+- Certificates and other secrets are now mounted in a way that is compatible
+  with Quay and Quay's Config App. #187
+- The operator now verifies the configuration for the Hostname, Redis, and
+  Postgres when Quay's configuration secret is changed. #177
+- Changed the default "From" email address used by Quay. #177
+- The Operator now uses the latest Project Quay image. #177
+- Changed the schema for managing external access in the CR. #164
+- Fixed a spelling error in log output. #169
+- Fixed the "Advanced Configuration" example. #163
```
[the remark this answers is not in the document]
Comment (kurtismullins) [reply CHANGELOG.md]: Did this one make it into 1.1.0? Which PR is that?

[the remark this answers is not in the document]
Comment (kurtismullins) [reply CHANGELOG.md]: Which PR adds support for >1 replicas? I thought that has always been a feature?

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**